app/src/home.js

```diff
@@ -242,7 +244,79 @@ export default function Home() {
     });
   }, [ ])
 
+  const handleSave = (e) => {
+    e.preventDefault()
+    saveSession()
+  }
+
+  async function createSession(data) {
+    const apiName = 'MTADemoAPI';
+    const path = 'createSession';
+    const myInit = { 
+    //   headers: { 
+    //     Authorization: `Bearer ${(await Auth.currentSession()).getIdToken().getJwtToken()}`,
+    //   },
+      response: true,
+      queryStringParameters: data
+    };
+
+    const result =  await API.post(apiName, path, myInit); 
+
+    return result;
+  }
 
+  const saveSession = () => {
+    const id = uuid();
```

Comment:
Please use uuidv4
https://www.npmjs.com/package/uuidv4

Reply:
Hi, I found the generate function in short-uuid can also generate a uniquer id with short length. https://github.com/aws-samples/medical-transcription-analysis/blob/eafae50489bbda5b2cd751d3679ef7a7ae8d11ce/app/src/home.js#L273